app/javascript/src/components/Dashboard/Contacts/Pane/constants.js

```diff
@@ -1,9 +1,10 @@
+import slugify from "slugify";
 import { buildSelectOption } from "utils";
 import * as yup from "yup";
 
 import { ROLES } from "components/Dashboard/constants";
 
-export const INITIAL_VALUES = {
+export const EMPTY_CONTACT = {
```

Comment:
Go through the latest comment I had put up in your last PR regarding this.

Reply:
Could you please check [this](https://github.com/vivek-viswam-rv/neeto-ui-challenge/pull/20#discussion_r984188139) comment?

Reply:
https://github.com/vivek-viswam-rv/neeto-ui-challenge/pull/20#discussion_r984215259